strapi/strapi
# Enhance blocks link validation

### What does it do?

Enable users to create mailto, tel and relative links in the blocks editor.
https://github.com/strapi/strapi/issues/18562

### Why is it needed?

Describe the issue you are solving.

### How to test it?

1. Create a new strapi entry for a type that contains a "blocks" field
2. Create a text link
3. enter one of the following values for the link
    - mailto:strapi@strapi.io
    - tel:+123456789
    - /strapi
5. Save the link and then save the entry
6. The link should be saved successfully.

### Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/18562

Let us know if this is related to any issue/pull request
